src/Components/SeatManager/SeatManager.js

```diff
@@ -79,7 +79,10 @@ class SeatManager extends Component {
                 onClick={(event) => this.handleSubmit(event)}>Add Seat</button>
             </form>
             <Link to={`/${this.props.currentUser.loginCode}/tables`}>
-              <h3 className='all-tables-button'>All Tables</h3>
+              <button className='all-tables-button'
+                disabled={this.props.currentTableOrder.length > 0}
```

Comment:
Can you just use this.props.currentTableOrder.length? I think it might return truthy.

Reply:
I think it would work, but I have similar code and prefer the comparison for a true or false vs a truthy or falsy